src/components/Summary.js

```diff
@@ -335,6 +335,44 @@ export default class Summary extends Component {
     return <div className="graph-placeholder"></div>;
   }
 
+  //New section to modify alert language
+  mapAlertContent(content) {
```

Comment:
@mldorr Maggie, the filtering list here can probably be moved to the Summary.json file.  But I think what you have here is good for now.